File: EGIS_Management/Domain/Entities/toims/CollectionSummary.cs
using System.ComponentModel.DataAnnotations;

namespace Domain.Entities
{
    public class CollectionSummary
    {
        [Key]
        public int Id { get; set; }
        public string Code { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public double BeginningBalance { get; set; }
        public double Amount { get; set; }
        public double Actual { get; set; }
        public int Month { get; set; }
        public int Year { get; set; }

    }
}

File: SPA19/EGIS/src/app/toims/models/collectionsummary.ts
export interface ICollectionSummary {
    id: number;
    code: string;
    description: string;
    beginningBalance : number;
    amount : number;
    actual : number;
    year : number;
    month : string;
 }

 export type createCollectionSummaryDto = Omit<ICollectionSummary,
    'code' | 'description' | 'beginningBalance' | 'amount' | 'actual' | 'year' | 'month'>;